packages/storybook/cxl-ui/cxl-inline-comment-context-menu/theme=cxl-faq.data.json

```diff
@@ -0,0 +1,211 @@
+[
```

Comment:
Is this a duplicate data file? Why not use existing data file?

Reply:
In case of any changes in JSON data structure for another story, this story won't crash.

Reply:
We'll deal with that when time comes, instead of early optimization.

Reply:
refactored in: https://github.com/conversionxl/aybolit/pull/86/commits/7a946a1285def6808798801aea8220d62f38a333